web/src/components/timeMachine/projectIntake/IntakeForm.vue

```diff
@@ -410,8 +409,13 @@ export default {
       this.$refs.projectFinance.onNextClicked();
 
       const riskAnalysis = this.getRiskAnalysis();
-      const scoreSum = riskAnalysis.map(o => o.score).reduce((a, c) => a + c);
-      const formData = assign({}, this.$refs.intakeBaseInfo.form, {
+      const mappedScores = riskAnalysis.map(o => o.score)
+      let scoreSum = 0;
+      if (mappedScores.length != 0 )
+      {
+      const scoreSum = mappedScores.reduce((a, c) => a + c);
```

Comment:
I have two small points about this one -

1. You should never use `const` after a variable has already been creating with `let` above.  You only ever use `const` OR `let` (or `var`) ONCE when you first declare the variable.
2. Please put your curly braces inline with the if (`if ( ...) {` ) and make sure to indent the line folllowing it.

```suggestion
           scoreSum = mappedScores.reduce((a, c) => a + c);
```

Reply:
BUT I'll accept it anyways because we're so busy :)